providers/nitro/nitro-helper/src/key_utils.rs

```diff
@@ -67,49 +65,49 @@ pub(crate) mod credential {
 /// Generates key and encrypts with AWS KMS at the given path
 /// TODO: generate in NE after this is merged https://github.com/aws/aws-nitro-enclaves-sdk-c/pull/55
 pub fn generate_key(
+    cid: u32,
+    port: u32,
     path: impl AsRef<Path>,
     region: &str,
     credentials: AwsCredentials,
     kms_key_id: String,
 ) -> Result<PublicKey, String> {
-    let credientials = KmsCredentials::from_keys(
-        credentials.aws_key_id,
-        &credentials.aws_secret_key,
-        Some(credentials.aws_session_token),
-    );
-    let mut csprng = OsRng {};
-    let keypair = Keypair::generate(&mut csprng);
-    let public = keypair.public;
-    let private_key = Zeroizing::new(keypair.secret.as_bytes().to_vec());
-    let privkey_blob = smithy_types::Blob::new(<Vec<u8> as AsRef<[u8]>>::as_ref(&private_key));
```

Comment:
should those AWS SDK dependencies (from Cargo.toml) be removed from the helper if they are not used?